kuka_rox_hardware_interface/include/mock/kuka/motion/external/external_control_mode.pb.h

```diff
@@ -82,8 +84,28 @@ inline const std::string& ExternalControlMode_Name(T enum_t_value) {
   static_assert(::std::is_same<T, ExternalControlMode>::value ||
     ::std::is_integral<T>::value,
     "Incorrect type passed to function ExternalControlMode_Name.");
-  return ::PROTOBUF_NAMESPACE_ID::internal::NameOfEnum(
-    ExternalControlMode_descriptor(), enum_t_value);
+  switch(enum_t_value)
+  {
+    case ExternalControlMode::POSITION_CONTROL:
+      control_mode_string = "POSITION_CONTROL";
```

Comment:
Why do we need control modes as strings? As I can see, this function is only used for logging and you can do the same with enums in less lines.

Reply:
Can you convert the enum name to string in a simple way, or what do you mean?
With the real proto lib, the descriptor is used to generate the string, but in mock I found it easier to just hardcode it to be able to build it

Reply:
No, I meant logging the enums as numbers, without converting them to strings. Mainly because the user already has to configure the control mode as an int. 

Probably not that important to address right now though, and the only thing you win with this is less lines of code.